src/phantasm-renderer/backend/assets/vertex_attrib_info.hh

```diff
@@ -51,35 +51,27 @@ constexpr format to_attribute_format()
 
     else if constexpr (tg::is_comp_like<T, 4, cc::int16>)
         return af::rgba16i;
-    else if constexpr (tg::is_comp_like<T, 3, cc::int16>)
-        return af::rgb16i;
     else if constexpr (tg::is_comp_like<T, 2, cc::int16>)
         return af::rg16i;
     else if constexpr (tg::is_comp_like<T, 1, cc::int16>)
         return af::r16i;
 
     else if constexpr (tg::is_comp_like<T, 4, cc::uint16>)
         return af::rgba16u;
-    else if constexpr (tg::is_comp_like<T, 3, cc::uint16>)
-        return af::rgb16u;
     else if constexpr (tg::is_comp_like<T, 2, cc::uint16>)
         return af::rg16u;
     else if constexpr (tg::is_comp_like<T, 1, cc::uint16>)
         return af::r16u;
 
     else if constexpr (tg::is_comp_like<T, 4, cc::int8>)
         return af::rgba8i;
-    else if constexpr (tg::is_comp_like<T, 3, cc::int8>)
```

Comment:
why are 3-comp formats removed?

Reply:
3-component formats do not exist in D3D12 and are only rarely supported in Vulkan (for example, rgb16f 8.5% vs rgba16f 100%, https://vulkan.gpuinfo.org/listformats.php)

Reply:
okay then let's leave them out of the backend. I'll have to see if we can do something about them in the frontend. For example, Jascha's stuff can build any component structure it wants internally.